build.sbt

```diff
@@ -68,7 +68,21 @@ libraryDependencies ++= Seq(
   "org.specs2" % "specs2_2.11" % "2.3.13" % "test"
 )
 
-scalacOptions ++= Seq("-unchecked", "-deprecation")
+// The OpenIE codebase generates compiler warnings like this:
+//
+//   Adapting argument list by creating a 2-tuple
+//   Adapting argument list by creating a 3-tuple
+//   implicit conversion method ... should be enabled
+//   method deserialize in object ... is deprecated
+//   possible missing interpolator
+//   reflective access of structural type member method close should be enabled
+//   side-effecting nullary methods are discouraged
+//   type parameter String defined in method apply shadows type String defined in object Predef. You may want to rename your type parameter, or possibly remove it.
+//
+// The code should be fixed to not have these problems. Until then, "-nowarn"
```

Comment:
I'm worried that "Until then" will become never if all the warnings are suppressed.  For style issues that might be fine, but for these compiler warnings we probably want to leave them around to force the issue of resolving them.

How many warnings are printed if you leave out `-nowarn`?